README.md

```diff
@@ -1,7 +1,44 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## Requirements
+
+- Node 18
+- yarn 3.2.4
+
 ## Getting Started
 
+### Initial setup
+
+1. Set the yarn version
+
+   ```bash
+   yarn set version 3.2.4
+   ```
+
+2. To get Kausal themes from the private registry when installing dependencies, create a `.yarnrc.yml` file with the following contents. Ask a teammate for the value of `npmAuthIdent`.
+
+   ```yml
+   npmScopes:
+   kausal:
+     npmAlwaysAuth: true
+     npmAuthIdent: <SECRET>
+     npmRegistryServer: 'https://npm.kausal.tech'
+   ```
+
+3. Configure yarn to install dependencies as regular `node_modules`
+
+   ```
+   yarn config set nodeLinker 'node-modules'
+   ```
+
+4. To run local development against the staging backend, create an `.env` file with the following env variable
+
+   ```
+   DEFAULT_GRAPHQL_API_URL=https://paths-api.staging.kausal.tech/v1/graphql/
```

Comment:
I would leave the URL out from here, because ideally this file would also serve as generic OSS instructions. Maybe formulate it the same way as with `npmAuthIdent` above?